android/build.gradle

```diff
@@ -10,7 +10,7 @@ buildscript {
   }
 
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.2.1'
+    classpath 'com.android.tools.build:gradle:7.4.2'
```

Comment:
i think this means minimum gradle 7.5 is supported right? https://developer.android.com/studio/releases/gradle-plugin

Reply:
yes, is there any concerns on bumping this for payments?

Reply:
I think this means we would stop supporting React Native versions that need below that, so .69 and below: https://github.com/facebook/react-native/blob/0.69-stable/template/android/gradle/wrapper/gradle-wrapper.properties

Reply:
do you need 7.4 specifically? or could we stay at `com.android.tools.build:gradle:7.2.2` for now?